### DISCUSSION TITLE
Change Scrollbar Size in Windows 11

### DISCUSSION THREAD

**Discussion by stani63** [ramensoftware/windhawk-mods#448] [Mod ideas]
Change Scrollbar Size (width, thickness) in Windows 11.

**Comment by coxymla** [comment]
This would be a godsend. I'm so tired of my scrollbars constantly morphing based on hover.

**Comment by amymor** [comment]
You can change it in registry:
HKEY_CURRENT_USER\Control Panel\Desktop\WindowMetrics
ScrollHeight
ScrollWidth

Here is a batch script that changes it from the default 15px to 18px ,you can change it to whatever you want (at least sign-out is required)
```
@echo off
:: https://www.elevenforum.com/t/change-scrollbar-size-in-windows-11.6465/
:: -15 is equal to 1 pixel, so -225 is equal to 15 pixels 
:: Range: -120 (smaller) to -1500 (larger)
:: Example values:
:: -225 = 15px
:: -270 = 18px
:: -300 = 20px
:: -330 = 22px
:: -375 = 25px

:-270=18px
:: Buttons Size
reg add "HKEY_CURRENT_USER\Control Panel\Desktop\WindowMetrics" /v "ScrollHeight" /t REG_SZ /d "-270" /f
:: Scrollbar Width
reg add "HKEY_CURRENT_USER\Control Panel\Desktop\WindowMetrics" /v "ScrollWidth" /t REG_SZ /d "-270" /f
pause
exit
```

**Comment by OrigamesPlayz** [comment]
This is actually sick! Im gonna try this!
